presto-main/src/main/java/com/facebook/presto/util/GraphvizPrinter.java

```diff
@@ -185,6 +185,22 @@ public static String printDistributed(SubPlan plan, Session session, FunctionAnd
 
         return output.toString();
     }
+    public static String printDistributedFromFragments(List<PlanFragment> allFragments, Session session, FunctionAndTypeManager functionAndTypeManager)
```

Comment:
Let's be consistent with the ordering of the argument functionAndTypeManager and session. In PlanPrinter constructor, the ordering was reversed. Same for all other occurrences.